includes/admin.form.inc

```diff
@@ -1564,21 +1597,32 @@ function _mods_id($mods_value = NULL, $pid = NULL) {
       }
       // Build the field suffix.
       if (strlen($mods_value) == 0) {
-        $build_string = 'For pid "' . $pid . '" <span style="color:red">Path cannot be blank</span>';
+        $build_string = t('For pid !pid: <span style="color:red">Path cannot be blank</span>', array(
+          '!pid' => $pid,
```

Comment:
Same question: is it safe to use `!placeholder`? Similarly for [L1606](https://github.com/Islandora/islandora_scholar/pull/299/commits/e5216a82ff3b166d0c8c98db21689fa39c7fa525#diff-0cdd19abe14e51fe9e7f061ec72c86d3R1606), [L1611](https://github.com/Islandora/islandora_scholar/pull/299/commits/e5216a82ff3b166d0c8c98db21689fa39c7fa525#diff-0cdd19abe14e51fe9e7f061ec72c86d3R1611), [L1616](https://github.com/Islandora/islandora_scholar/pull/299/commits/e5216a82ff3b166d0c8c98db21689fa39c7fa525#diff-0cdd19abe14e51fe9e7f061ec72c86d3R1616) and [L1617](https://github.com/Islandora/islandora_scholar/pull/299/commits/e5216a82ff3b166d0c8c98db21689fa39c7fa525#diff-0cdd19abe14e51fe9e7f061ec72c86d3R1617).